common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -37,6 +37,10 @@
   private static Method sCleanerCleanMethod;
   private static Method sByteBufferCleanerMethod;
 
+  public static int byteToInt(byte b) {
+    return (int) b & 0xFF;
```

Comment:
I think cast has higher precedence but probably not needed. I can remove the case and resubmit.
The CI is failing because some import order issue in ReadOnlyJournal.java, is this a known issue?